src/components/header/_macro.spec.js

```diff
@@ -6,523 +6,348 @@ import axe from '../../tests/helpers/axe';
-    ariaLabel: 'Main menu',
-    currentPath: '#1',
-    currentPageTitle: 'Guidance',
-    itemsList: [
-      {
-        title: 'Home',
-        url: '#0',
-      },
-      {
-        title: 'Guidance',
-        url: '#1',
-      },
-    ],
-    toggleNavigationButton: {
-      text: 'Menu',
-      ariaLabel: 'Toggle main menu',
-    },
-    subNavigation: {
-      id: 'sub-nav',
-      overviewURL: '#overview',
-      overviewText: 'Overview',
-      ariaLabel: 'Section menu',
-      currentPath: '#1',
-      itemsList: [
-        {
-          title: 'Sub nav item 1',
-          url: '#0',
-          classes: 'custom-class-sub-item-1',
-          id: 'sub-item-1',
-        },
-        {
-          title: 'Sub nav item 2',
-          url: '#1',
-          classes: 'custom-class-sub-item-2',
-          id: 'sub-item-2',
-        },
-      ],
-    },
-  },
-};
-
-const EXAMPLE_HEADER_NAVIGATION_WITH_SITESEARCHAUTOSUGGEST = {
-  navigation: {
-    id: 'main-nav',
-    ariaLabel: 'Main menu',
-    currentPath: '#home',
-    itemsList: [
-      {
-        title: 'Home',
-        url: '#home',
-      },
-      {
-        title: 'Guidance',
-        url: '#0',
-      },
-    ],
-    toggleNavigationButton: {
-      text: 'Menu',
-      ariaLabel: 'Toggle main menu',
-    },
-  },
-  siteSearchAutosuggest: {
-    label: 'label',
-    instructions: 'Use up and down keys to navigate.',
-    ariaYouHaveSelected: 'You have selected',
-    ariaMinChars: 'Enter 3 or more characters for suggestions.',
-    minChars: 3,
-    ariaResultsLabel: 'Country suggestions',
-    ariaOneResult: 'There is one suggestion available.',
-    ariaNResults: 'There are {n} suggestions available.',
-    ariaLimitedResults: 'Type more characters to improve your search',
-    moreResults: 'Continue entering to improve suggestions',
-    resultsTitle: 'Suggestions',
-    resultsTitleId: 'country-of-birth-suggestions',
-    noResults: 'No suggestions found.',
-    typeMore: 'Continue entering to get suggestions',
-    language: 'en-gb',
-  },
-};
-
-describe('macro: header', () => {
-  describe('mode: basic', () => {
-    it('passes jest-axe checks', async () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_BASIC));
-
-      const results = await axe($.html());
-      expect(results).toHaveNoViolations();
+import {
+  EXAMPLE_HEADER_BASIC,
+  EXAMPLE_SERVICE_LINKS_CONFIG,
+  EXAMPLE_HEADER_SERVICE_LINKS_MULTIPLE,
+  EXAMPLE_HEADER_SERVICE_LINKS_SINGLE,
+  EXAMPLE_HEADER_LANGUAGE_CONFIG,
+  EXAMPLE_HEADER_NAVIGATION_WITH_SUBNAVIGATION_CONFIG,
+  EXAMPLE_HEADER_NAVIGATION_WITH_SITESEARCHAUTOSUGGEST,
+} from './_test_examples';
+
+describe('FOR: Header', () => {
+  describe('GIVEN: Params: none', () => {
+    describe('WHEN: All params are at default state', () => {
+      const $ = cheerio.load(
+        renderComponent('header', {
+          ...EXAMPLE_HEADER_BASIC,
+        }),
+      );
+      test('THEN: Jest-Axe tests pass', async () => {
```

Comment:
```suggestion
      test('THEN: jest-axe tests pass', async () => {
```
This library is lowercase